include/posix.h

```diff
@@ -81,6 +79,8 @@ typedef int ino_t; /* FIXME: should be unsigned long long to encode id_t? */
 typedef int nlink_t;
 typedef int blksize_t;
 typedef int blkcnt_t;
+typedef long long off64_t;
+typedef off64_t off_t;
```

Comment:
We were planning on moving these definitions to kernel's include and using it in libphoenix

Reply:
I agree. To be done in another PR.